### PR TITLE
[EXT-50] New extension: Zus Health

### PR DESCRIPTION
**Pull request by kubiak85**
null

<sub>[EXT-60](https://linear.app/awell/issue/EXT-60/[ext-50]-new-extension-zus-health)</sub>

### CODE REVIEW

**Review by rahulkeerthi — CHANGES_REQUESTED** [review]

File: extensions/zusHealth/webhooks/getResource.ts
[the remark this answers is not in the document]
Comment (kubiak85) [reply extensions/zusHealth/webhooks/getResource.ts:1]: @rahulkeerthi Two webhooks with the same functionality?

[the remark this answers is not in the document]
Comment (rahulkeerthi) [reply extensions/zusHealth/webhooks/getResource.ts:1]: @kubiak85 sorry, I did say two different things -- let me try again!

- yes, two webhooks, patientAdmitted and patientDischarged
- in patientAdmitted, we do the same until we have the Encounter fetched then we check the status. If it is not an admit event (using the logic above) then we do nothing, and if it is, we start a care flow (onSuccess)
- in patientDischarged, same as above but we check if it is a discharge event (using conditions above) and same, do nothing if it is not and start a care flow if it is a discharge

File: extensions/zusHealth/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Zus Health
+description: Zus is the only shared health data platform designed to accelerate healthcare data interoperability by providing easy-to-use patient data at the point of care via API, embedded components, and direct EHR integrations.
+---
+
+# Zus Health
+
+[ZUS Health](https://zushealth.com/) is the only shared health data platform designed to accelerate healthcare data interoperability by providing easy-to-use patient data at the point of care via API, embedded components, and direct EHR integrations.
```
Comment (rahulkeerthi) [inline extensions/zusHealth/README.md:8]:
```suggestion
[Zus Health](https://zushealth.com/) is the only shared health data platform designed to accelerate healthcare data interoperability by providing easy-to-use patient data at the point of care via API, embedded components, and direct EHR integrations.
```

**Review by rahulkeerthi — CHANGES_REQUESTED** [review]

File: extensions/zusHealth/webhooks/patientAdmitted.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+      const api = makeAPIClient(settings)
+      const resource = await api.getResource(resourceId)
+
+      if (resource.status === 'in-progress' && !isNil(resource.period.start)) {
+        await onSuccess({
+          data_points: {
+            resourceId,
+            UPID: payload.resource.UPID,
+            userId: payload.ownerId,
```
Comment (rahulkeerthi) [inline extensions/zusHealth/webhooks/patientAdmitted.ts:65]: see next comment

**Review by rahulkeerthi — APPROVED** [review]
